tests/test_analysis.py

```diff
@@ -1,100 +1,167 @@
 import inspect
+import importlib
+import json
+import sys
+
+# import delphi.analysis.sensitivity.variance_methods as methods
```

Comment:
Ok for now if you're still testing things out, but we should remove old commented-out code before merging into master.